packages/core/src/components/dropdown/dropdown.scss

```diff
@@ -33,7 +33,7 @@
 }
 
 :host(.overflow) {
-  max-height: 50vh;
+  max-height: calc(50vh - 50px); 
```

Comment:
Maybe we can use here a variable which makes it looks less like a magic number `50px`